### PR TITLE
mkfile: add page

### PR DESCRIPTION
**Pull request by glenacota**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/osx/mkfile.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# mkfile
+
+> Create one or more empty files of any size.
+
+- Create an empty file of 15 kilobytes:
+
+`mkfile -n {{15k}} {{file_name}}`
+
+- Create a file of a given size and unit (bytes, kB, MB, GB):
```
Comment (sbrl) [inline pages/osx/mkfile.md]: Does the `k` of `kB` have to be lowercase? All the others are uppercase.

File: pages/osx/mkfile.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+`mkfile -n {{size}}{{b|k|m|g}} {{file_name}}`
+
+- Create two files of 4 megabytes:
```
Comment (sbrl) [inline pages/osx/mkfile.md]:
```suggestion
- Create 4 megabyte files:
```

Perhaps this will read better?

File: pages/osx/mkfile.md (hunk 1 above)
Comment (glenacota) [reply pages/osx/mkfile.md]: 👍

File: pages/osx/mkfile.md (hunk 2 above)
Comment (glenacota) [reply pages/osx/mkfile.md]: I'm not sure.. I'm not a native English speaker, but I would interpret it as "create 4 files of 1 megabytes each"
If it's only me, I'll be happy to apply your suggestion

Comment (sbrl) [reply pages/osx/mkfile.md]: `Create 2 files of 4 megabytes each` would work too :-)

**Pull request merged**